Corona-Warn-App/src/main/AndroidManifest.xml

```diff
@@ -65,7 +65,8 @@
             android:name=".ui.main.MainActivity"
             android:exported="false"
             android:screenOrientation="portrait"
-            android:theme="@style/AppTheme.Main" />
+            android:theme="@style/AppTheme.Main"
+            android:windowSoftInputMode="adjustResize" />
```

Comment:
Why the new flag?

Reply:
Ok I see, it comes from the ContactDiary Activity. How does effect input in the "non-diary" app?

Where do we have keyboard input that needs to be tested due to this

* TAN input
* :thinking:

Reply:
TAN works nicely, also symptoms screens, as well as onboarding.